src/pydvl/influence/frameworks/twice_differentiable.py

```diff
@@ -1,9 +1,11 @@
 from abc import ABC
+from collections import namedtuple
 from typing import Callable, Generic, List, Sequence, Tuple, TypeVar
 
 TensorType = TypeVar("TensorType", bound=Sequence)
 ModelType = TypeVar("ModelType")
 DeviceType = TypeVar("DeviceType")
+iHVPResult = namedtuple("iHVPResult", ["x", "info"])
```

Comment:
I think it's better to use the `NamedTuple` class from the `typing` package because it allows the use of type hints for the fields.

```suggestion
class iHVPResult(NamedTuple):
   x: TensorType
   info: Dict[str, Union[Any, TensorType]]
```